### PR TITLE
Add how-to doc on displaying user date timezones

### PR DESCRIPTION
**Pull request by kahlouie**
## Description

As we fill out forms, we'd like to make sure all the information makes sense to our end users. To this end we've decided that dates should be local to the user generating the form. This documents that for future devs that face a similar question of how do we display dates with the complexity of timezones. This is how DDS has said other government projects approach dates.

I've asked each of the team leads to review so that each team can remember the decision when this inevitably comes up in the future again. This decision is about display, not about date/time storage.

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* https://www.pivotaltracker.com/story/show/163433031

### CODE REVIEW

**Comment by kahlouie** [comment]
Added @jim. Good point.

File: docs/how-to/display-dates-and-times.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# How to display dates and times
+
+Timezones are hard to do correctly, especially factoring in daylight savings time and users in multiple locations using our app.
+
+## Displaying dates in forms
+
+Other government projects use the date that the person filling the form would have used (their date in their local timezone). This is what we'll continue doing. The official format in the office and TSP app so far has been DD-Mon-YYYY (29-Mar-2018, as an example). In the service member interfaces, we use the date format MM/DD/YYYY (03/29/2018). This should be what we default to unless otherwise stated. We don't currently store a user's local timezone, so unless we make changes to do that, we should send the timezone for the user back to the server from the client. Note that this will get more complicated when we add OCONUS moves, but in the meantime it should suffice.
```
Comment (akostibas) [inline docs/how-to/display-dates-and-times.md]: > We don't currently store a user's local timezone, so unless we make changes to do that, we should send the timezone for the user back to the server from the client.

I'm not sure I understand this. We want to be storing dates in the database as UTC, which means that user-displayed times need to be converted either on the client or the server. Is this sentence suggesting where that conversion should take place?

I don't see a need to store the user's timezone as the client can make the appropriate UTC->localtime conversions. That 'storing the user's timezone on the server' is mentioned at all makes me wonder if I'm missing some point.

Comment (kahlouie) [reply docs/how-to/display-dates-and-times.md]: @akostibas We currently have a need for displaying the current date (not a stored date) as considered local to the user. This is most common for forms that used to be filled out manually. In this case, we need to know what timezone the user is in currently or get the date directly from the browser. Here's a pr that @mkrump worked on that uses this: https://github.com/transcom/mymove/pull/1670. I think invoices hit a similar question, but punted on finding the answer at the moment.

Comment (jim) [reply docs/how-to/display-dates-and-times.md]: I think a bit of missing context is that when we're generating PDFs, we need to know the current user's current time (including tz) on the server, where the PDF is generated, and not in the client, where it's easy to display a UTC time in local time.

**Pull request merged**